.github/workflows/tests.yaml

```diff
@@ -119,6 +119,9 @@ jobs:
       - name: Run end-to-end tests
         run: ./test/e2e_test.sh
 
+      - name: Run pkcs11 end-to-end tests
```

Comment:
Can you add this as a dedicated job in https://github.com/sigstore/cosign/blob/main/.github/workflows/e2e-tests.yml?